File: 0049-group-anagrams/0049-group-anagrams.cpp
class Solution {
public:
    vector<vector<string>> groupAnagrams(vector<string>& strs) {
        map<vector<int>, vector<string>> vv;
        vector<vector<string>> ans;
        for(string s : strs){
            vector<int> v(26);
            for(char c:s){
                v[c-'a']++;
            }
            vv[v].push_back(s);
        }
        for(auto it:vv){
            ans.push_back(it.second);
        }
        return ans;
    }
};

File: 0049-group-anagrams/0049-group-anagrams.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
        d = {}
        for i in strs:
            k = tuple(sorted(i))
            if k in d:
                d[k].append(i)
            else:
                d[k] = []
                d[k].append(i)
        # print(d.values())
        return d.values()